pom.xml

```diff
@@ -64,6 +64,13 @@
     <properties>
         <jenkins.version>1.642.3</jenkins.version>
```

Comment:
BTW you may as well just go to 2.7.x already. Then we just have two versions of Groovy to test: baseline, and `jenkins.version=2.60.2` (i.e., 2.48+ IIRC).